app/src/shared/components/ElevatedContainer.jsx

```diff
@@ -0,0 +1,51 @@
+// @flow
+
+import React, { useState, useEffect } from 'react'
+import styled, { css } from 'styled-components'
+import { TOOLBAR_SHADOW, MD } from '$shared/utils/styled'
+import useInt from '$shared/hooks/useInt'
+
+export const Root = styled.div`
+    ${({ elevated, offset }) => (offset === 0 || !!elevated) && css`
```

Comment:
Shadow is always there if offset is 0 (default) or elevated flag is up (when scrolled down enough).